pkg/endpointmanager/manager.go

```diff
@@ -460,6 +464,35 @@ func (mgr *EndpointManager) AddEndpoint(owner regeneration.Owner, ep *endpoint.E
 	return nil
 }
 
+func (mgr *EndpointManager) AddHostEndpoint(ctx context.Context, owner regeneration.Owner,
+	proxy endpoint.EndpointProxy, allocator cache.IdentityAllocator, reason string, nodeName string) (err error) {
+	ep := endpoint.CreateHostEndpoint(owner, proxy, allocator)
+	if err := mgr.AddEndpoint(owner, ep, reason); err != nil {
+		return err
+	}
+
+	epLabels := labels.Labels{}
+	epLabels.MergeLabels(labels.LabelHost)
+
+	if k8s.IsEnabled() {
+		// Retrieve k8s labels.
+		if k8sNode, err := k8s.GetNode(k8s.Client(), nodeName); err != nil {
```

Comment:
This node will be the node where `cilium-agent` is running, right?

Reply:
Yes, but `cilium-agent` runs on all nodes managed by Cilium, right?

Reply:
Yes, just trying to be sure.